src/plugins/Clock/index.ts

```diff
@@ -0,0 +1,176 @@
+        description: "IANA Time Zone Code for slot two",
+        type: OptionType.STRING,
+        default: "America/Toronto",
+        restartNeeded: true,
+    },
+    slotThreeEnabled: {
+        description: "Whether to show slot three",
+        type: OptionType.BOOLEAN,
+        default: false,
+        restartNeeded: true,
+    },
+    slotThreeLabel: {
+        description: "Label for slot three",
+        type: OptionType.STRING,
+        default: "London",
+        restartNeeded: true,
+    },
+    slotThreeZone: {
+        description: "IANA Time Zone Code for slot three",
+        type: OptionType.STRING,
+        default: "Europe/London",
+        restartNeeded: true,
+    },
+    slotFourEnabled: {
+        description: "Whether to show slot four",
+        type: OptionType.BOOLEAN,
+        default: false,
+        restartNeeded: true,
+    },
+    slotFourLabel: {
+        description: "Label for slot four",
+        type: OptionType.STRING,
+        default: "Sydney",
+        restartNeeded: true,
+    },
+    slotFourZone: {
+        description: "IANA Time Zone Code for slot four",
+        type: OptionType.STRING,
+        default: "Australia/Sydney",
+        restartNeeded: true,
+    },
+});
+
+export default definePlugin({
+    name: "Clock",
+    description: "Adds a customizable clock with timezones",
+    authors: [Devs.Tnixc],
+    settings,
+    async start() {
+        const { use12Hour, showSeconds, seperator, rightAlign } = settings.store;
+        const { slotOneEnabled, slotOneLabel, slotOneZone } = settings.store;
+        const { slotTwoEnabled, slotTwoLabel, slotTwoZone } = settings.store;
+        const { slotThreeEnabled, slotThreeLabel, slotThreeZone } = settings.store;
+        const { slotFourEnabled, slotFourLabel, slotFourZone } = settings.store;
+
+        const createClockOptions = (hour12: boolean, timeZone: string): Intl.DateTimeFormatOptions => ({
+            timeZone: timeZone,
+            hour12: hour12,
+            hour: "2-digit",
+            minute: "2-digit",
+            ...(showSeconds && { second: "2-digit" }),
+        });
+
+        const clockOneOptions = createClockOptions(use12Hour, slotOneZone);
+        const clockTwoOptions = createClockOptions(use12Hour, slotTwoZone);
+        const clockThreeOptions = createClockOptions(use12Hour, slotThreeZone);
+        const clockFourOptions = createClockOptions(use12Hour, slotFourZone);
+
+        const panels = document.querySelector(".panels__58331") as HTMLElement;
+
+        if (!panels) return;
+
+        const clockContainer = document.createElement("div");
+        clockContainer.classList.add("vc-clock-container");
+
+        if (rightAlign) clockContainer.classList.add("right-align");
+
+        panels?.appendChild(clockContainer);
+        if (slotOneEnabled) {
+            var slotOneClock = document.createElement("p");
+            slotOneClock.classList.add("vc-clock");
+            clockContainer?.appendChild(slotOneClock);
```

Comment:
This is literally never null/undefined
```suggestion
            clockContainer.appendChild(slotOneClock);
```